user_model.ml

```diff
@@ -1,10 +1,13 @@
 module Rng = Mirage_crypto_rng
 
 type token = {
+  name : string;
   token_type : string;
   value : string;
   expires_in : int;
   created_at : Ptime.t;
+  last_access : Ptime.t;
+  usage_count : int;
       (* In seconds, so after 1 hour would be 3600 seconds of inactivity *)
```

Comment:
I don't understand the comment here...

What I see is a created_at and expires_in, so I'd assume it is valid from created_at until created_at + expires_in (in seconds?).

Would be great to have that documented. the comment here is a bit off/unattached now.

Reply:
This comment slipped in. I don't think it's relevant anymore.